### PR TITLE
Add h1 header

### PR DESCRIPTION
**Pull request by ordinaryRoman**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import { Header } from './components/Header/Header';
```
Comment (LenaYork) [inline src/App.js:1]: According to the style guide each component folder should contain index.js file.

**Comment by LenaYork** [comment]
Importation should be implemented with that index.js reference

File: src/App.js (hunk 1 above)
Comment (ordinaryRoman) [reply src/App.js]: Now header folder contains it

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+h1 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 48px;
+    font-style: italic;
+    text-align: center;
+    color: darkolivegreen;
+}
```
Comment (LenaYork) [inline src/components/Header/Header.css:7]: Empty line required

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+h1 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 48px;
+    font-style: italic;
```
Comment (LenaYork) [inline src/components/Header/Header.css:4]: cool

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -3,6 +3,6 @@ import "./Header.css";
 
 export function Header() {
     return(
-        <div></div>
+        <h1>TODO list</h1>
     )
 }
```
Comment (LenaYork) [inline src/components/Header/Header.js:8]: Empty line required

File: src/components/Header/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export * from './Header';
```
Comment (LenaYork) [inline src/components/Header/index.js:1]: Empty line required

**Pull request merged**